source/_integrations/ituran.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: Ituran
+description: Instructions on how to add Ituran to Home Assistant.
+ha_category:
+  - Car
+  - Device Tracker
+  - Presence detection
+ha_release: '2025.01'
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@shmuelzon'
+ha_domain: ituran
+ha_platforms:
+  - device_tracker
+ha_integration_type: integration
+---
+
+The **Ituran** {% term integration %} allows you to retrieve information from your Ituran-equipped vehicle. It pulls information from the Ituran web service regarding the vehicle's location.
+
+{% note %}
+The information is pulled every 5 minutes from the Ituran web service; however, the data is updated at intervals determined by Ituran based on whether the vehicle is stationary or not.
+{% endnote %}
+
+## Prerequisites
+
+You must have an Ituran account for use with the Ituran APP ([Android](https://play.google.com/store/apps/details?id=com.ituran.forall)/[iOS](https://apps.apple.com/app/id1227756834)). All devices that work with the app, should work with this integration as well.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+ID or passport number:
+  description: Your government ID or passport number used to sign-up with Ituran.
+Mobile phone number:
+  description: The mobile phone number used to sign-up with Ituran. A one-time-password will be sent to this number.
+Mobile ID:
+  description: A unique identifier used for registration with Ituran services. Leave this blank to generate a new ID automatically. Only provide a value if you want to share an existing Mobile ID from another device (e.g., your phone's Ituran app) to keep both devices active simultaneously.
```

Comment:
Can be removed now

Reply:
It's weird that you still see this as I already removed it 🤷‍♂️